travis-ci/managers/debian.sh

```diff
@@ -36,16 +37,17 @@ for phase in "${PHASES[@]}"; do
             if [[ "$phase" = "RUN_CLANG" ]]; then
                 ENV_VARS="-e CC=clang -e CXX=clang++"
             fi
-            docker exec $ENV_VARS -it $CONT_NAME mkdir build
-            $DOCKER_EXEC make CFLAGS='-Werror' -C ./src -B OBJDIR=../build
+            $DOCKER_EXEC mkdir build
+            docker exec $ENV_VARS -it $CONT_NAME make CFLAGS="${CFLAGS}" -C ./src -B OBJDIR=../build
```

Comment:
Nit: any reason not to use `$DOCKER_EXEC $ENV_VARS` on line 41?

Reply:
Need to slightly modify `DOCKER_EXEC` var for that:
`
DOCKER_EXEC="${DOCKER_EXEC:-docker exec -it $CONT_NAME}"
`
vs.
`docker exec $ENV_VARS -it $CONT_NAME`

`$DOCKER_EXEC $ENV_VARS` concatenation would have another meaning.

Reply:
That's what I was trying to figure out :) OK, if order of arguments matters here, then it should be fine to go as is. Thanks!

Reply:
Resolved by adding `docker_exec` function: https://github.com/libbpf/libbpf/pull/40/commits/0ba9ce75796df3fe70771d0d25af7b1162adef55#diff-6e5ee78613021bcff97487ad55a3748eR17

Reply:
Also gcc-8 tests are added for Debian: https://travis-ci.org/libbpf/libbpf/builds/526211312